### PR TITLE
Fix adding of Atlantis

### PR DESCRIPTION
**Pull request by Code0x58**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -194,10 +194,10 @@ RUN set -e \
     && echo "$KIND_SHA256  kind" | sha256sum -c \
 # atlantis
     && wget -q -O /tmp/atlantis.zip https://github.com/runatlantis/atlantis/releases/download/v${ATLANTIS_VERSION}/atlantis_linux_amd64.zip \
-    && echo "$ATLANTIS_SHA256  atlantis.zip" | sha256sum -c \
-    && unzip atlantis.zip -d /usr/local/bin \
+    && echo "$ATLANTIS_SHA256  /tmp/atlantis.zip" | sha256sum -c \
```
Comment (primeroz) [inline Dockerfile]: can you just do a cd /tmp instead of this ?

Comment (Code0x58) [reply Dockerfile]: Ah! I mistakenly put this code between the last block for _kind_ and the `cd -` that should be at the end of it.

About half of the other blocks to get tools use `/tmp/…` from within `/tmp/`. There's probably some tidying that could be done on the blocks to make them a little more consistent and :golf:

Comment (primeroz) [reply Dockerfile]: right

File: Dockerfile [hunk 2]
```diff
@@ -192,13 +192,13 @@ RUN set -e \
     && cd /usr/local/bin \
     && chmod +x /usr/local/bin/kind \
     && echo "$KIND_SHA256  kind" | sha256sum -c \
+    && cd - \
 # atlantis
     && wget -q -O /tmp/atlantis.zip https://github.com/runatlantis/atlantis/releases/download/v${ATLANTIS_VERSION}/atlantis_linux_amd64.zip \
-    && echo "$ATLANTIS_SHA256  atlantis.zip" | sha256sum -c \
-    && unzip atlantis.zip -d /usr/local/bin \
+    && echo "$ATLANTIS_SHA256  /tmp/atlantis.zip" | sha256sum -c \
+    && unzip /tmp/atlantis.zip -d /usr/local/bin \
```
Comment (primeroz) [inline Dockerfile]: but i can't still see /tmp in every command ?

**Pull request merged**